external/source/exploits/CVE-2022-1471/README.md

```diff
@@ -0,0 +1,19 @@
+# Overview
+The Java file contained within will load and execute a Metasploit payload. It's intended to be loaded as part of the
+exploit for CVE-2022-1471 which is a YAML deserialization vulnerability within the snakeyaml project.
+
+See https://bitbucket.org/snakeyaml/snakeyaml/issues/561/cve-2022-1471-vulnerability-in for more information.
+
+## Compiling
+Compile the Java source file using `javac -cp path/to/metasploit-payloads/data/java MyScriptEngineFactory.java`.
```

Comment:
Maybe add a line about how to generate the `data` directory as well, that references [Building the Java and Android Meterpreter](https://github.com/rapid7/metasploit-payloads/blob/master/java/README.md#building-the-java-and-android-meterpreter)?
```suggestion
Compile the Java source file using `javac -cp path/to/metasploit-framework/data/java MyScriptEngineFactory.java`.
```